net/amneziawg-tools/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2016-2019 Jason A. Donenfeld <Jason@zx2c4.com>
+# Copyright (C) 2016 Baptiste Jonglez <openwrt@bitsofnetworks.org>
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=amneziawg-tools
+
+PKG_VERSION:=1.0.20250706
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/amnezia-vpn/amneziawg-tools.git
+PKG_MIRROR_HASH:=b5ebcfd260ed578420002aa60ec95326558cead64ebc4ab655888ff4b75687a1
+PKG_SOURCE_VERSION:=2b593ee0860e2ecc7cfcbfab4261194256329048
+
+PKG_MAINTAINER:=Gregory Gullin <garuwex@gmail.com>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_PATH:=src
+MAKE_VARS += PLATFORM=linux
+
+define Package/amneziawg-tools
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=VPN
+  URL:=https://amnezia.org/
+  TITLE:=AmneziaWG userspace control program
+
+  DEPENDS:= \
+	  +!BUSYBOX_CONFIG_IP:ip \
+	  +!BUSYBOX_CONFIG_FEATURE_IP_LINK:ip \
+	  +kmod-amneziawg
+endef
+
+define Package/amneziawg-tools/description
+  Amnezia VPN — simple and free app to run a self-hosted VPN with
+	high privacy requirements.
+
+  This package provides the userspace control program for AmneziaWG,
+  `amneziawg`, a netifd protocol helper, and a re-resolve watchdog script.
```

Comment:
Again, mixing tabs and spaces. And it's not clear what the purpose of the `amneziawg` bit is here. Is it referring to some part of this package? I don't see anything with that name in the `install` section.